.papr.yml

```diff
@@ -0,0 +1,40 @@
+branches:
+  - master
+
+host:
+  distro: fedora/26/cloud
+
+required: true
+
+packages:
+    - git
+    - iptables
+    - runc
+    - golang
+    - golang-github-cpuguy83-go-md2man
+    - libseccomp-devel
+    - gpgme-devel
+    - device-mapper-devel
+    - libassuan-devel
+    - glibc-static
+    - glib2-devel
+    - bats
+    - btrfs-progs-devel
+    - bzip2
+    - device-mapper-devel
+    - findutils
+    - git
+    - glib2-devel
+    - gnupg
+    - golang
+    - gpgme-devel
+    - libassuan-devel
+    - libseccomp-devel
+    - libselinux-devel
```

Comment:
It looks like papr doens't like the duplicates.

Reply:
no, makes it very angry 😀 s'okay, my own damn fault (again).